src/dataServices/dynamoDbBundleServiceHelper.test.ts

```diff
@@ -290,3 +297,279 @@ describe('populateBundleEntryResponseWithReadResult', () => {
         expect(actualResult).toEqual(expectedResult);
     });
 });
+
+describe('processBatchRequests', () => {
```

Comment:
Add a test to deliberately check the errors were mapped back to the correct item? e.g. Two success and two fail in one request and the errors were mapped back correctly.